### DISCUSSION TITLE
Please, tell me your favorite ramen shop in Kansai!!

### DISCUSSION THREAD

**Discussion by h1karu-s** [NAIST-SE/AttractingContributors#561] [General]
### Discussed in https://github.com/NAIST-SE/AttractingContributors/discussions/519

<div type='discussions-op-text'>

<sup>Originally posted by **hikaru-sh** June 17, 2022</sup>
I'd like to know your favorite ramen shop in Kansai
あなたの好きな関西にあるラーメン屋が知りたいです。

If it is convenient,  please add your favorite ramen shop to the List of Hobby.
よろしければ、好きなラーメン屋を「List of Hobby」に追加してください。

https://github.com/hikaru-sh/AttractingContributors-Ramen

__My favorite ramen shop__
ramen mituba (ラーメン屋みつ葉)
Human beings everybody noodles.（人類みな麺類)</div>

**Comment by ani-hovhannisyan** [comment]
I contributed, please can you do it back?
My project: https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle
Thanks in advance

**Comment by raux** [comment]
I am converting this issue to a discussion.
